docs-ref-conceptual/includes/cli-install-linux-dnf.md

```diff
@@ -2,18 +2,18 @@
 author: chasecrum
 ms.author: chasecrum
 manager: mamccrea
-ms.date: 08/01/2023
+ms.date: 08/1/2023
 ms.topic: include
 ms.custom: devx-track-azurecli
 ---
 
 ## Before you begin
 
-- Azure CLI [2.38.0](/cli/azure/release-notes-azure-cli#july-05-2022) (released on 07/05/2022) is the last version supported on RHEL 7 and Centos 7 and will continue to receive security fixes until the end of life of the operating systems. Microsoft will not provide additional updates or bug fixes for these distributions.
+- Azure CLI [2.38.0](/cli/azure/release-notes-azure-cli#july-05-2022) (released on July 5, 2022) is the last version supported on RHEL 7 and Centos 7 and will continue to receive security fixes until the end of life of the operating systems. Microsoft isn't providing other updates or bug fixes for these distributions.
 
-- Azure CLI and the Linux [Install script](/cli/azure/install-azure-cli-linux?pivots=script) will continue to be developed and supported by Microsoft. Packages provided by Red Hat and CentOS repositories are maintained and supported by their respective distributions.
+- Microsoft continues to develop and support Azure CLI and the Linux [Install script](/cli/azure/install-azure-cli-linux?pivots=script). Red Hat and CentOS distributions provide packages, while maintaining and supporting their respective repositories. 
```

Comment:
> Red Hat and CentOS distributions provide packages, while maintaining and supporting their respective repositories.

This sentence is hard to understand to me.

Reply:
@jiasli, This sentence had to be reworded to meet Acrolinx standards.